src/components/InteractiveQuestionView.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Flex, Group, Modal } from "@mantine/core"
+import { InteractiveQuestion } from "@metabase/embedding-sdk-react"
+
+import { ThemedButton } from "./ThemedButton"
+import { CustomIcon } from "./CustomIcon"
+import { useReducer } from "react"
+import { useDisclosure } from "@mantine/hooks"
+
+// TODO: add the "breakout" view once we've separated the breakout from summarization in the SDK
+type QuestionView = "viz" | "filter" | "summary" | "editor"
+
+interface Props {
+  isSaveEnabled?: boolean
+}
+
+export const InteractiveQuestionView = (props: Props) => {
+  const { isSaveEnabled = false } = props
+
+  const [view, changeView] = useReducer(
+    (view: QuestionView, nextView: QuestionView) =>
+      view !== "viz" && view === nextView ? "viz" : nextView,
+    "viz",
+  )
+
+  const [isSaveModalOpen, { open: openSaveModal, close: closeSaveModal }] =
+    useDisclosure()
+
+  return (
+    <Box>
+      <Flex
+        w="100%"
+        justify="space-between"
+        pb={14}
+        direction={{ base: "column", sm: "row" }}
+        rowGap="sm"
+      >
+        <Group gap="xs">
+          <InteractiveQuestion.BackButton />
```

Comment:
How do you get this button to render? I don't couldn't figure out how.

Reply:
@WiNloSt Good callout, I just realized that the back button is hidden if we do not have `onNavigateBack` specified. We don't mention this in the docs either - let's update the doc to indicate this.